dev-libs/libsodium/libsodium-1.0.16.ebuild

```diff
@@ -16,16 +16,19 @@ IUSE="+asm minimal static-libs +urandom cpu_flags_x86_sse4_1 cpu_flags_x86_aes"
 
 PATCHES=( "${FILESDIR}"/${PN}-1.0.10-cpuflags.patch )
 
-src_prepare() {
+multilib_src_prepare() {
```

Comment:
There's no multilib version of `src_prepare`, so revert this change and your problem should be gone ;-).

Reply:
Whoops ;D Fixed in newest commit. Thanks!